packaging/StopWatch.spec

```diff
@@ -1,6 +1,6 @@
 Name:          StopWatch
 Version:       %{version}
-Release:       1%{?dist}
+Release:       %{buildnumber}%{?dist}
```

Comment:
Have you tested this?  I thought you would have to add  build number to the version string
Ref: https://github.com/LogRhythm/StopWatch/blob/master/CMakeLists.txt
```SET_TARGET_PROPERTIES(${LIBRARY_TO_BUILD} PROPERTIES LINKER_LANGUAGE CXX SOVERSION ${VERSION})```

Cool if the rpm build system just adds it. 

How does  one version change from another?
I.e. a re-build from version 146 (libFileIO.so.1.146)
What would the next library be named?